static/app/components/events/eventTags/eventTagsTree.spec.tsx

```diff
@@ -160,9 +147,6 @@ describe('EventTagsTree', function () {
     expect(anchorLink.href).toContain(
       `/mock-pathname/?rd=show&rdRelease=${releaseVersion}&rdSource=release-version-link`
     );
-    expect(reposRequest).toHaveBeenCalled();
-    expect(releasesRequest).toHaveBeenCalled();
-    expect(deploysRequest).toHaveBeenCalled();
     const dropdown = screen.getByLabelText('Tag Actions Menu');
```

Comment:
api calls are now no longer made, which is correct since the card doesn’t render unless interacted with.